cerberus-api-tests/src/main/resources/json-schema/negative/ownership-required-permissions-error.json

```diff
@@ -0,0 +1,49 @@
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "http://example.com/example.json",
+  "type": "object",
+  "properties": {
+    "error_id": {
+      "$id": "/properties/error_id",
+      "type": "string",
+      "title": "The Error_id Schema.",
+      "description": "An explanation about the purpose of this instance.",
+      "default": "",
+      "examples": [
+        "713019ae-515e-4716-ba04-225d57fbb42a"
+      ]
+    },
+    "errors": {
+      "$id": "/properties/errors",
+      "type": "array",
+      "items": {
+        "$id": "/properties/errors/items",
+        "type": "object",
+        "properties": {
+          "code": {
+            "$id": "/properties/errors/items/properties/code",
+            "type": "integer",
+            "title": "The Code Schema.",
+            "description": "An explanation about the purpose of this instance.",
```

Comment:
Is this default text?  

Also, below.

Reply:
Yes, this is a generated file. I didn't see any real value in populating all of the values.

Let me know if you disagree.

Reply:
Yes, this is a generated file. I didn't see any real value in populating all of the values.

Let me know if you disagree.